common/djangoapps/course_modes/views.py

```diff
@@ -153,16 +154,24 @@ def get(self, request, course_id, error=None):
 
         enterprise_learner_data = enterprise_api.get_enterprise_learner_data(site=request.site, user=request.user)
         if enterprise_learner_data:
-            context["show_enterprise_context"] = True
-            context["partner_names"] = partner_name = course.display_organization \
-                if course.display_organization else course.org
-            context["enterprise_name"] = enterprise_learner_data[0]['enterprise_customer']['name']
-            context["username"] = request.user.username
-            organizations = organization_api.get_course_organizations(course_id=course.id)
-            if organizations:
-                context["partner_names"] = ' and '.join([
-                    org.get('name', partner_name) for org in organizations
-                ])
+            is_course_in_enterprise_catalog = enterprise_api.is_course_in_enterprise_catalog(
+                site=request.site,
+                course_id=course_id,
+                user=request.user,
+                enterprise_catalog_id=enterprise_learner_data[0]['enterprise_customer']['catalog']
```

Comment:
If there is None against the catalog, the api will raise the exception which we are catching in enterprise_helpers.py and return `is_course_in_enterprise_catalog` would be False